Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -437,6 +448,92 @@ public String getKeysInSharedPrefs(String key) {
         return KeyValueStore.read(juspayServices, key, "__failed");
     }
 
+    @JavascriptInterface
+    public void addCarousel(String stringifyArray,String id){
+        activity.runOnUiThread(() -> {
+            try {
+
+                ViewPager2 viewPager2 = new ViewPager2(context);
```

Comment:
Add null check for activity. as it is nullable. try to reduce the try catch block

Reply:
We need try catch block here as we are extracting data from JSON